packager/media/crypto/encryption_handler.cc

```diff
@@ -390,7 +390,8 @@ void EncryptionHandler::EncryptBytes(const uint8_t* source,
   DCHECK(source);
   DCHECK(dest);
   DCHECK(encryptor_);
-  CHECK(encryptor_->Crypt(source, source_size, dest));
+  CHECK(encryptor_->Crypt(source, source_size, dest, &source_size));
```

Comment:
Why the extra parameter?

Reply:
I was curious about this as well, but I figured I'd start with @zdanek's changes and we can iterate from there.

Reply:
Reverted the changes, as it seemed this wasn't needed, since the 3 param version calls the 4 param version internally.